tests/ibc-defi/osmosis/osmosis-init.sh

```diff
@@ -17,16 +17,14 @@ CHAIN_ID="osmosis"
 # Node
 osmosisd init --chain-id "$CHAIN_ID" testing
 
-# User
+# Config
+sed -i $SED_EXT 's/"stake"/"uosmo"/' "$HOME/.osmosisd/config/genesis.json"
+sed -i $SED_EXT 's|laddr = "tcp://127.0.0.1:26657"|laddr = "tcp://0.0.0.0:26657"|g' "$HOME/.osmosisd/config/config.toml"
+sed -i $SED_EXT 's|address = "localhost:9090"|address = "0.0.0.0:9090"|g' "$HOME/.osmosisd/config/app.toml"
+
 osmosisd keys add osmosis-user --keyring-backend=test
 
 # Genesis
 osmosisd add-genesis-account "$(osmosisd keys show osmosis-user -a --keyring-backend=test)" 2000000000uosmo
 osmosisd gentx osmosis-user 500000000uosmo --keyring-backend=test --chain-id "$CHAIN_ID"
 osmosisd collect-gentxs
-
-jq '.app_state["gov"]["voting_params"]["voting_period"]="10s"' "$HOME/.osmosisd/config/genesis.json" > "$HOME/.osmosisd/config/tmp_genesis.json" && \
-  mv "$HOME/.osmosisd/config/tmp_genesis.json" "$HOME/.osmosisd/config/genesis.json"
```

Comment:
Don't we need this update?

Reply:
That's unnecessary in the context of standalone IBC tests, therefore safe to remove.